### DISCUSSION TITLE
Port without Traeffic

### DISCUSSION THREAD

**Discussion by RobinDev03** [Music-Bot-for-Jitsi/Jimmi#147] [Q&A]
Hey,

short question:
Which port should i publish, if i don#t want to use traeffic?
I already have an proxy installed on my docker-host and want to use this one.

**Comment by p-fruck** [comment]
This usecase is not supported. You might want to look into the setup of the [version 1](https://github.com/Music-Bot-for-Jitsi/Jimmi/tree/v1.0.0) of Jimmi. The current version was a POC that derived from a `Software Engineering II` lesson and is not going to be maintained because of the massive resource overhead. Once there is enough time to maintain this project I will pick up the original version 1 release which is a lot less complex and resource intensive.
